src/support_sphere_db/src/ssdb/models/user_profile.py

```diff
@@ -0,0 +1,18 @@
+import uuid
+from sqlmodel import Field, Relationship
+from ssdb.models import User
+from ssdb.models.abstract_public_schema_model import AbstractPublicSchemaModel
+
+
+class UserProfile(AbstractPublicSchemaModel, table=True):
```

Comment:
Please add a small docstring that describe this table saying that this table contains metadata information about a user?

Reply:
updated.